permutations.r

```diff
@@ -1,22 +1,49 @@
-Packages <- c("gtools", "readr", "tibble", "dplyr")
 
-lapply(Packages, library, character.only = TRUE)
+################################################################################
+
+# WHAT IS THIS SCRIPT FOR?
```

Comment:
Something I need to get better about myself is starting scripts with a section that tells the reader what the script is for. It's often good to have it in a big block like this. I've seen other styles as well.